test/left.jl

```diff
@@ -0,0 +1,62 @@
+using Test, LinearMaps
+using LinearAlgebra: mul!
+
+# y'*L is an exception to the left multiplication rule that makes a WrappedMap
+
+@static if VERSION ≥ v"1.4.0"
+
+function left_tester(L::LinearMap{T}) where {T}
```

Comment:
❤️ I love this "test function" style, and realized recently in the other PR when searching for a certain "test pattern" for quite a while that all our tests should be rewritten like that.